roles/ceph-haproxy/meta/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+galaxy_info:
+  author: Gui Hecheng
+  description: Config HAProxy
+  license: Apache
+  min_ansible_version: 2.3
```

Comment:
include_tasks have been introduced in ansible 2.4 so the min_ansible_version should reflect that.

Reply:
@dsavineau thanks for pointing the exact version out, and I shall add firewalld rules and selinux configs.